specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/OperationalInsights.json

```diff
@@ -0,0 +1,429 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition."
+          },
+          "404": {
+            "description": "Not found the specific data export."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataExport": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DataExportProperties",
+          "description": "data export properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The top level data export resource container.",
+      "x-ms-azure-resource": true
+    },
+    "DataExportListResult": {
+      "description": "Result of the request to list data exports.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataExport"
+          },
+          "description": "List of data export instances within a workspace.."
+        }
+      }
+    },
+    "DataExportProperties": {
+      "properties": {
+        "dataExportId": {
+          "type": "string",
+          "description": "The data export rule ID."
+        },
+        "allTables": {
+          "type": "boolean",
+          "description": "When ‘true’, all workspace's tables are exported."
+        },
+        "tableNames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An array of tables to export, for example: [“Heartbeat, SecurityEvent”]."
+        },
+        "destination": {
+          "$ref": "#/definitions/Destination",
+          "description": "destination properties.",
+          "x-ms-client-flatten": true
+        },
+        "enable": {
+          "type": "boolean",
+          "description": "The data export rule creation time."
+        },
+        "createdDate": {
+          "type": "string",
+          "description": "The latest data export rule modification time."
+        },
+        "lastModifiedDate": {
+          "type": "string",
+          "description": "Date and time when the export was last modified."
+        }
+      },
+      "description": "Data Export properties."
+    },
+    "Destination": {
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The destination resource ID. This can be copied from the Properties entry of the destination resource in Azure."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource",
+          "enum": [
+            "StorageAccount",
+            "EventHub"
+          ]
+        },
```

Comment:
i suppose type can be derived from the resourceId..

Also, you may consider making destination as an array if you envision supporting export to multiple destinations

Reply:
We want the type explicitly to avoid customer's mistakes.
It we will always one destination to export.